src/brogue/Monsters.c

```diff
@@ -52,15 +52,15 @@ void mutateMonster(creature *monst, short mutationIndex) {
 // Allocates space, generates a creature of the given type,
 // prepends it to the list of creatures, and returns a pointer to that creature. Note that the creature
 // is not given a map location here!
+// TODO: generateMonster is convenient, but probably it should not add the monster to
+// any global lists. The caller can do this, to avoid needlessly moving them elsewhere.
```

Comment:
I might agree, otherwise the return state is inconsistent, with the creature on the chain/list but not placed